networktables2/common.py

```diff
@@ -36,7 +36,7 @@ def keys(self):
         :returns: the list of keys
         """
         with self.mutex:
-            return self.namedEntries.keys().copy()
```

Comment:
Because the object you get from ```keys()``` doesn't actually have a ```copy()``` attribute. I had originally thought that the ```keys()``` alone would work, but I just checked and the python docs say otherwise. Therefore I added a generator expression that emulates ```copy()``` in the latest commit.

When you think it is good, I will rebase it to a single commit.